shell/platform/windows/win32_window.cc

```diff
@@ -74,7 +81,7 @@ LRESULT CALLBACK Win32Window::WndProc(HWND const window,
                      reinterpret_cast<LONG_PTR>(cs->lpCreateParams));
 
     auto that = static_cast<Win32Window*>(cs->lpCreateParams);
-    that->current_dpi_ = that->dpi_helper_->GetDpi(window);
+
```

Comment:
Remove the blank line; the line above is specifically for the next line, so they should be grouped.